models/src/course.rs

```diff
@@ -13,15 +16,61 @@ pub struct CourseSummary {
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
 pub struct OfferingSummary {
     pub year: i16,
-    pub term: String
+    pub term: String,
 }
 
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
 #[serde(transparent)]
 pub struct OfferingSummaries(Vec<OfferingSummary>);
 
-#[cfg(feature = "sqlx")]
-use sqlx::{Database, Postgres, Type};
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
+pub struct CourseDetail {
+    // TODO: Store CourseSummary as a field
```

Comment:
We should come back and try to wrap + decode `CourseSummary` from sqlx rather than duplicating it's fields.